resources/windows/wix/service.pxp-agent.wxs.erb

```diff
@@ -0,0 +1,61 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+  <Fragment>
+    <ComponentGroup Id="Service_PXPAGENT">
+      <ComponentRef Id="Service_NSSM" />
```

Comment:
Or, wait, do these need to be consistent from pftw? If that's the case, then ignore that last comment.